src/integrations/watchers/indexable-post-watcher.php

```diff
@@ -264,8 +264,11 @@ protected function update_relations( $post ) {
 		$related_indexables = $this->get_related_indexables( $post );
 
 		foreach ( $related_indexables as $indexable ) {
-			$indexable->object_last_modified = \max( $indexable->object_last_modified, $post->post_modified_gmt );
-			$indexable->save();
+			// Ignore everything that is not an actual indexable.
+			if ( \is_a( $indexable, Indexable::class ) ) {
```

Comment:
We already mentioned that there's no need for this check, according to [this](https://github.com/Yoast/wordpress-seo/pull/19629#issuecomment-1396830622).

But we can still leave this in as a nice measure for the future (I think that the impact check entry we added for hierarchies already cover how to test this part as well)